Makefile

```diff
@@ -68,7 +68,7 @@ build: $(BINS)
 
 $(BIN_DIR)/%-$(BIN_ARCH)$(BIN_SUFFIX): $(SRC) go.mod go.sum
 	mkdir -p $(BIN_DIR)
-	$(GOBUILD) -ldflags="-s -w -X main.version=$(GIT_REF) -X main.gitSha=$(GIT_SHA)" \
+	$(GOBUILD) -buildvcs=false -ldflags="-s -w -X main.version=$(GIT_REF) -X main.gitSha=$(GIT_SHA)" \
```

Comment:
Is this causing any trouble? I'm not well familiar with the `buildvcs`flag, but I understand it should just include vcs info into the build. I'm not against disabling it, just that I would like to understand the reason.

Reply:
Yep, macos was refusing to build

Reply:
macOS binary in the GitHub CI? Or local? Do you have the error message?

I'm wondering what the underlying issue is, and if we can keep this enabled - I would prefer to resolve/understand the cause to introducing this.

Reply:
@stepanstipl can browse the error here: https://github.com/doitintl/kube-no-trouble/runs/8050840335?check_suite_focus=true#step:5:83

Reply:
Looks like error with the safe directory (again: 🤦 ) - I've tried to add the git command go is using to get the vcsinfo directly in the build, see #344 and https://github.com/doitintl/kube-no-trouble/runs/8058460382?check_suite_focus=true#step:5:22, fails with the following:

```
fatal: detected dubious ownership in repository at '/__w/kube-no-trouble/kube-no-trouble'
To add an exception for this directory, call:

	git config --global --add safe.directory /__w/kube-no-trouble/kube-no-trouble
Error: Process completed with exit code 128.
```

I though this would be taken care of by using the `set-safe-directory: true` argument on the checkout action, but apparently not. I want to look into this as to why not.

Reply:
yep, its the git issue again. The set safe-directory to true should do it.

Reply:
@stepanstipl I seem to have fixed it! Have a look at https://github.com/doitintl/kube-no-trouble/pull/337/files#diff-71cabc4177e41ea8f15a89eb65398fa8187739f68a2762706d84885cd8b2ddc3R55, I just ran the safe command again.

Reply:
Might be due to different users per task?